features/step_definitions/list-streams.rb

```diff
@@ -0,0 +1,15 @@
+Given('that I have signed-in') do
+    pending # Write code here that turns the phrase above into concrete actions
```

Comment:
I believe we have a `/` page, so we can change here to  `visit / `
And maybe put `visit /` in all of them just to check the behavior and not have broking tests

Reply:
I don't think that's a good solution, I did some research and found out that removing the `--strict` tag from cucumber.yml makes so that `pending` returns 0.

Reply:
I'll take a look to see if the `@wip` tag solves our problems.

Reply:
> I don't think that's a good solution, I did some research and found out that removing the `--strict` tag from cucumber.yml makes it so that `pending` returns 0.

That seems to be a good solution